tools/javadoc/tests.bzl

```diff
@@ -97,13 +186,17 @@ def _root_packages_test_case(name):
     )
 
 def analysis_test_suite(name):
+    _deps_test_case(name = name + "_dep")
     _group_test_case(name = name + "_group")
+    _links_test_case(name = name + "_links")
     _root_packages_test_case(name = name + "_root_packages")
 
     native.test_suite(
         name = name,
         tests = [
+            ":%s_dep_test" % name,
```

Comment:
You don't need the colon, so you could create variables you can reuse here and when you define the test case.

Reply:
I don't understand. In any event, I don't _have_ to parameterize by `name` (it's unlikely this package will have multiple suites where uniqueness matters), so I'll just keep it simple and stick to literals like `deps_test` and `group_test`.